Nightly s3 job is failing

The `release-s3` job in the `nightly` workflow [is failing](https://github.com/timberio/vector/runs/762264450?check_suite_focus=true):

```

...

 Waiting for 30 seconds before running the verifications
Verifying https://packages.timber.io/vector/nightly/2020-06-11/vector-x86_64-unknown-linux-musl.tar.gz
Verifying https://packages.timber.io/vector/nightly/latest/vector-x86_64-unknown-linux-musl.tar.gz
Verifying https://packages.timber.io/vector/nightly/latest/vector-x86_64-unknown-linux-gnu.tar.gz
/dev/fd/63 /tmp/tmp.48472MJwF7/vector-x86_64-unknown-linux-musl.tar.gz differ: byte 5, line 1
make: *** [release-s3] Error 1
Makefile:238: recipe for target 'release-s3' failed
##[error]Process completed with exit code 2.
```

The files are being uploaded, but the verification step is not passing.

--- a/scripts/release-s3.sh
+++ b/scripts/release-s3.sh
@@ -83,7 +83,7 @@ elif [[ "$CHANNEL" == "latest" ]]; then
   done
   verify_artifact \
     "https://packages.timber.io/vector/latest/vector-x86_64-unknown-linux-gnu.tar.gz" --fail \
-    "$td/vector-x86_64-unknown-linux-musl.tar.gz"
+    "$td/vector-x86_64-unknown-linux-gnu.tar.gz"
 fi
 
 #

